Common/include/CConfig.hpp

```diff
@@ -856,7 +856,8 @@ class CConfig {
   nMu_Temperature_Ref,           /*!< \brief Number of species reference temperature for Sutherland model. */
   nMu_S,                         /*!< \brief Number of species reference S for Sutherland model. */
   nThermal_Conductivity_Constant,/*!< \brief Number of species constant thermal conductivity. */
-  nPrandtl_Lam,                  /*!< \brief Number of species laminar Prandtl number. */
+  nPrandtl_Lam,                  /*!< \brief Number of species 
+  addDoubleOption("FREESTREAM_TURB2LAMVISCRATIO", TurbIntensityAndViscRatioFreeStream[1], 10.0); Prandtl number. */
```

Comment:
Was this intentionally commented, or...? 
```suggestion
  nPrandtl_Lam,                  /*!< \brief Number of species laminar Prandtl number. */
  addDoubleOption("FREESTREAM_TURB2LAMVISCRATIO", TurbIntensityAndViscRatioFreeStream[1], 10.0); /*!<\brief  Freestream mu_turb to mu_lam viscosity ratio */
```

Reply:
This is just a wrong copy-paste from me. It should not be there in the first place.